### PR TITLE
Add wrapper for package_python_2_7_statsmodels_0_6_1

### PR DESCRIPTION
**Pull request by bebatut**

### CODE REVIEW

File: packages/package_python_2_7_statsmodels_0_6_1/tool_dependencies.xml [hunk 1]
```diff
@@ -0,0 +1,50 @@
...
+                    </package>
+                </action>
+                <action type="set_environment">
+                    <environment_variable action="append_to" name="PYTHONPATH">$INSTALL_DIR</environment_variable>
+                    <environment_variable action="set_to" name="PYTHONPATH_STATSMODEL">$INSTALL_DIR</environment_variable>
+                    <environment_variable action="set_to" name="STATSMODEL_ROOT_PATH">$INSTALL_DIR</environment_variable>
+                </action>
+            </actions>
+        </install>
+        <readme>Simple, fast implementation of Fisher’s exact test.</readme>
```
Comment (yhoogstrate) [inline packages/package_python_2_7_statsmodels_0_6_1/tool_dependencies.xml]: The statsmodels package can do a bit more

**Comment by yhoogstrate** [comment]
Ready for testing via TTS from my side :+1:

**Comment by bebatut** [comment]
Hi,

Is it possible to push it, at least on TTS, please?

Thanks 

Bérénice

**Comment by bgruening** [comment]
@bebatut welcome back :)
I updated the packages. Maybe we should push packages automatically via travis.

**Comment by bebatut** [comment]
@bgruening Thanks :)

**Pull request merged**